tools/replay/rp_visualization.py

```diff
@@ -33,7 +33,8 @@ def visualize(addr):
     if sm.recv_frame['radarState']:
       plot_lead(sm['radarState'], lid_overlay)
     liveTracksTime = sm.logMonoTime['liveTracks']
-    maybe_update_radar_points(sm['liveTracks'], lid_overlay)
+    if sm['liveTracks'] is not None and hasattr(sm['liveTracks'], 'trackList'):
```

Comment:
this should work
```suggestion
    if sm.updated['liveTracks']:
```

Reply:
and with that, can we remove the "maybe_"?

Reply:
Haha agreed, done.